packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/hybrid_search.py

```
@@ -1,40 +1,33 @@
 
+from ..config import SearchConfig
 from ..utils.logging import LoggingConfig
-
 from .nlp.spacy_analyzer import SpaCyQueryAnalyzer
-
-# Enhanced search components
-from .enhanced.intent_classifier import IntentClassifier, AdaptiveSearchStrategy, SearchIntent
+from .enhanced.intent_classifier import IntentClassifier, AdaptiveSearchStrategy
 from .enhanced.knowledge_graph import DocumentKnowledgeGraph
 from .enhanced.topic_search_chain import (
     TopicSearchChainGenerator, 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)